.github/workflows/run-archiver.yml

```diff
@@ -7,27 +7,77 @@ on:
     - cron: "21 8 1 * *" # 8:21 AM UTC, first of every month
 
 jobs:
-  archive-run:
+  archive-run-eia:
     defaults:
       run:
         shell: bash -l {0}
     strategy:
       matrix:
         dataset:
-          - eia176
           - eia191
           - eia757a
           - eia860
           - eia860m
           - eia923
-          - epacems
           - eia_bulk_elec
 
       fail-fast: false
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install Conda environment using mamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: environment.yml
+          cache-environment: true
+          condarc: |
+            channels:
+            - conda-forge
+            - defaults
+            channel_priority: strict
+
+      - name: Log the conda environment
+        run: |
+          conda info
+          conda list
+          conda config --show-sources
+          conda config --show
+          printenv | sort
+
+      - name: Run archiver for ${{ matrix.dataset }}
+        env:
+          ZENODO_SANDBOX_TOKEN_UPLOAD: ${{ secrets.ZENODO_SANDBOX_TOKEN_UPLOAD }}
+          ZENODO_SANDBOX_TOKEN_PUBLISH: ${{ secrets.ZENODO_SANDBOX_TOKEN_PUBLISH }}
+          EPACEMS_API_KEY: ${{ secrets.EPACEMS_API_KEY }}
+          ZENODO_TOKEN_UPLOAD: ${{ secrets.ZENODO_TOKEN_UPLOAD }}
+          ZENODO_TOKEN_PUBLISH: ${{ secrets.ZENODO_TOKEN_PUBLISH }}
+        run: |
+          pudl_archiver --datasets ${{ matrix.dataset }} --summary-file ${{ matrix.dataset }}_run_summary.json
+
+      - name: Upload run summaries
+        if: always()
+        id: upload_summaries
+        uses: actions/upload-artifact@v4
+        with:
+          name: run-summaries-${{ matrix.dataset }}
+          path: ${{ matrix.dataset }}_run_summary.json
+
+  archive-run-cems:
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
```

Comment:
non-blocking: you *could* rip out this whole `strategy` block and just hard-code `epacems` instead of using `${{ matrix.dataset }}` but 🤷 this works and is clear enough.

Reply:
I imagine if we do start regularly archiving EQR it'll go in here as well, so I'll just leave it.